README.md

```diff
@@ -40,5 +45,15 @@
   > bundle:start <net.staticsnow:nexus-repository-apt ID>
   ```
 
+#### Permanent Install
+
+For more permanent installs of the nexus-repository-apt plugin, follow these instructions:
+
+* Copy the bundle (nexus-repository-apt-1.0.2.jar) into <nexus_dir>/deploy
```

Comment:
A more comprehensive approach may be to put the bundle in `<nexus_dir>/system/net/staticsnow/nexus-repository-apt/1.0.2/nexus-repository-apt-1.0.2.jar`. Then add the following lines to `<nexus_dir>/system/com/sonatype/nexus/assemblies/nexus-oss-feature/3.x.y/nexus-oss-feature-3.x.y-features.xml`
```
--- nexus-oss-feature-3.x.y-features-orig.xml	2017-03-20 13:42:03.000000000 -0500
+++ nexus-oss-feature-3.x.y-features.xml	2017-03-20 13:43:43.000000000 -0500
@@ -18,2 +18,3 @@
         <feature prerequisite="false" dependency="false">nexus-repository-rubygems</feature>
+        <feature prerequisite="false" dependency="false">nexus-repository-apt</feature>
         <feature prerequisite="false" dependency="false">nexus-repository-gitlfs</feature>
@@ -185,2 +186,6 @@
     </feature>
+    <feature name="nexus-repository-apt" description="net.staticsnow:nexus-repository-apt" version="1.0.2">
+        <details>net.staticsnow:nexus-repository-apt</details>
+        <bundle>mvn:net.staticsnow/nexus-repository-apt/1.0.2</bundle>
+    </feature>
 </features>
```